gwpy/astro/range.py

```diff
@@ -436,7 +437,13 @@ def inspiral_range(psd, snr=8, mass1=1.4, mass2=1.4, fmin=None, fmax=None,
     # return the sensitive distance metric
     return units.Quantity(
         inspiral.cosmo.luminosity_distance(z_hor) if horizon else
-        range_func(f[frange], psd.value[frange], z_hor=z_hor, H=inspiral),
+        range_func(
+            f[frange],
+            psd.value[frange],
+            z_hor=z_hor,
+            H=inspiral,
+            detection_snr=snr
```

Comment:
Please include the trailing comma on this line, which will mean that any future changes adding more keywords will not have to modify this line.

```suggestion
            detection_snr=snr,
```